test/test-applications/integrations/Directory.Build.props

```diff
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="..\..\..\Test.Common.props" />
```

Comment:
I'll just leave one last comment instead of every `Directory.Build.props`  😜 

I think for these `props` files that import another `props` file, we should use `$(MSBuildThisFileDirectory)`. IIRC, paths like `..\..\..\Test.Common.props` are relative to the project that included this file, not relative to this file. Using a relative path like this means all projects that include this file need to be nested at the same depth.

Reply:
Here's a good summary of how MSBuild treats imports: https://github.com/MicrosoftDocs/visualstudio-docs/issues/4287#issuecomment-550506186

Basically:
- Project imports with relative paths are resolved from the location of that file
- ItemGroup includes with relative paths are resolved from the original projection location

As a result:
- I left the relative paths in the Import statements alone because those are resolved as expected
- I modified the `<Compile Include="..\GlobalSuppressions.cs"/>` to `<Compile Include="$(MSBuildThisFileDirectory)GlobalSuppressions.cs"/>`

Reply:
TIL, thanks! Resolving comment.